fv3gfs-physics/fv3gfs/physics/_config.py

```diff
@@ -139,7 +139,7 @@ def from_namelist(cls, namelist: Namelist) -> "PhysicsConfig":
             vi_fac=namelist.vi_fac,
             vr_fac=namelist.vr_fac,
             de_ice=namelist.de_ice,
-            layout=namelist.layout,
+            layout=namelist.layout,  # type: ignore
```

Comment:
(same as above) Is this type: ignore still needed, now that you fixed the problem with passing lists vs tuples on init?